### PR TITLE
changes to census address page

### PR DESCRIPTION
**Pull request by leannecook**
### What is the context of this PR?
i have removed the 'building' text field and added a description to address line 1.

### How to review 
Check census_household.json for the address page

### CODE REVIEW

File: data/en/census_household.json [hunk 1]
```diff
@@ -42,14 +42,8 @@
...
                     "answers": [{
-                            "id": "building",
-                            "label": "Building",
-                            "mandatory": false,
-                            "type": "TextField"
-                        },
-                        {
                             "id": "address-line-1",
-                            "label": "Address Line 1",
+                            "label": "Address Line 1 (Including your house number)",
```
Comment (jonnyshaw89) [inline data/en/census_household.json]: `house number` or should this be `house name or number`

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: data/en/census_household.json (hunk 1 above)
Comment (leannecook) [reply data/en/census_household.json]: i have updated with your suggestion :)

File: tests/integration/census/test_census_household_submission_data.py [hunk 2]
```diff
@@ -17,14 +17,6 @@ def test_census_household_data_matches_census_individual(self):
     def get_expected_submission_data():
         expected_downstream_data = [
             {
-                'answer_id': 'building',
```
Comment (jonnyshaw89) [inline tests/integration/census/test_census_household_submission_data.py:19]: You also need to remove the answer from https://github.com/ONSdigital/eq-survey-runner/blob/9d9ac4a41b98eaa126c4a851867f9f3abcd4af07/tests/integration/census/test_census_household_submission_data.py#L1181

**Comment by shahi645** [comment]
@leannecook just need to update the test and then this looks good

File: tests/integration/census/test_census_household_submission_data.py (hunk 2 above)
Comment (leannecook) [reply tests/integration/census/test_census_household_submission_data.py:19]: done :)

**Review by jonnyshaw89 — APPROVED** [review]

**Review by shahi645 — APPROVED** [review]
Squishy-squashy, mergy-rebasey

**Pull request merged**